src/app/ipv/middleware.js

```diff
@@ -64,8 +64,13 @@ async function fetchUserDetails(req) {
   return generateUserDetails(userDetailsResponse, req.i18n);
 }
 
-async function handleJourneyResponse(req, res, action) {
+async function handleJourneyResponse(req, res, action, currentPage = "") {
```

Comment:
Is this default value useful - could it just default to `undefined`?

Reply:
Yeah I thought it was worth showing that this is optional, it always feels a bit wrong to me to just ignore a parameter with no override